tests/e2e/e2e_suite_test.go

```diff
@@ -134,7 +96,7 @@ var _ = BeforeSuite(func() {
 		// (no further bootstrap is required)
 		By("Upgrading the operator only", func() {
 			kubectl.DeleteNamespace("cattle-rancheros-operator-system")
```

Comment:
Is there any reasons why there is no error check in some lines in that code?

Reply:
No specific reason I think, probably a leftover

Reply:
Ok, I will add some with the next commit.

Reply:
Done!